target/scripts/startup/setup-stack.sh

```diff
@@ -1100,12 +1100,33 @@ function _setup_postfix_vhost
   _create_postfix_vhost
 }
 
-function _setup_inet_protocols
+function _setup_postfix_inet_protocols
 {
   _notify 'task' 'Setting up POSTFIX_INET_PROTOCOLS option'
   postconf -e "inet_protocols = ${POSTFIX_INET_PROTOCOLS}"
 }
 
+function _setup_dovecot_inet_protocols
+{
+  local PROTOCOL
+
+  _notify 'task' 'Setting up DOVECOT_INET_PROTOCOLS option'
+
+  # https://dovecot.org/doc/dovecot-example.conf
+  if [[ ${DOVECOT_INET_PROTOCOLS} == "ipv4" ]]
+  then
+    PROTOCOL='*' # IPv4 only
+  elif [[ ${DOVECOT_INET_PROTOCOLS} == "ipv6" ]]
+  then
+    PROTOCOL='[::]' # IPv6 only
+  else
+    # Unknown value, panic.
+    dms_panic__invalid_value 'DOVECOT_INET_PROTOCOLS' "${DOVECOT_INET_PROTOCOLS}"
+  fi
```

Comment:
With the new default value of `all`, this needs small adjustments, or am I overlooking something here?:)

Reply:
You do, see [here](https://github.com/casperklein/archive_docker-mailserver/blob/03d887a0c7c9f03754467c7b0f28a9126ec0435a/target/scripts/start-mailserver.sh#L114).

Reply:
I see :D